UnityProject/Assets/Scripts/Objects/Machines/Griddle.cs

```diff
@@ -351,5 +351,10 @@ public override void PowerStateUpdate(PowerState state)
 		}
 
 		#endregion
+
+		public void Dispose()
+		{
+			audioLoopLock?.Dispose();
```

Comment:
This was added by me in #9533

Is there not a scan happening on new PRs to catch when issues are introduced?

Reply:
We had codacy scanning enabled before, but it kept reporting false positives.

Reply:
Im his new codacy system now.